api_client/templates/fortipam.py

```diff
@@ -0,0 +1,223 @@
+LOGIN = """
+{
+    "path": "/XX/YY/ZZ/AUTH",
+    "method": "POST",
+    "body": "magic=175805987&username={{ username }}&password={{ password }}&4Tredir=%2F&4Tmthd=0&domain=local"
+}
+"""
+
+REDIRECT = """
```

Comment:
what is it?

Reply:
redirect? thats how they do login after login post, here is to follow the redirect